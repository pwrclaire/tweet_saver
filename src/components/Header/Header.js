import React from "react";
import styled from "styled-components";

const HeaderContainer = styled.h1`
  text-align: center;
  font-family: "Open Sans", sans-serif;
`;

function Header() {
  return (
    <HeaderContainer>
      TWEET SAVER{" "}
      <img src="https://img.icons8.com/fluent/48/000000/twitter.png" alt="" />
    </HeaderContainer>
  );
}

export { Header };
