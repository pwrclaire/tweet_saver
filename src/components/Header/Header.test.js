import React from "react";
import { Header } from "./Header";
import renderer from "react-test-renderer";

describe("Testing Header component", () => {
  test("should render", () => {
    const component = renderer.create(<Header />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
