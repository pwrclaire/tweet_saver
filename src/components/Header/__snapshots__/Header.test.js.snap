// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header component should render 1`] = `
<h1
  className="sc-AxjAm jatRZr"
>
  TWEET SAVER
   
  <img
    alt=""
    src="https://img.icons8.com/fluent/48/000000/twitter.png"
  />
</h1>
`;
